### ISSUE TITLE
1.7.10: Single payment interface throws error when no open invoices

### ISSUE THREAD

**Issue by ehuelsmann** [ledgersmb/LedgerSMB#4529]
When selecting a vendor from "Cash > Payment" which has no open payments, clicking "Continue" results in:

```plain
Can't locate object method "to_output" via package "0" (perhaps you forgot to load "0"?) at lib/LedgerSMB/Scripts/payment.pm line 1177.
```

**Comment by ehuelsmann** [comment]
Add test comment.

**Comment by ehuelsmann** [comment]
Fixed through 8a74df5
